applications/search/src/pages/search-page/results-tabs/results-tabs.component.jsx

```diff
@@ -15,8 +15,14 @@ export const ResultsTabs = props => {
   return (
     <div className="row">
       <div className="col-12 col-lg-8 offset-lg-4">
-        <ul className="search-results-tabs">
-          <li className={location.pathname === '/' ? 'li-active' : ''}>
+        <ul className="search-results-tabs d-flex align-items-center justify-content-between">
+          <li
+            className={
```

Comment:
Jeg tror det er samme som `cx('d-flex justify-content-center',{'li-active':location.pathname === '/'})`

Reply:
Ja, hvis jeg lager tre ulike const for de tre variantene av location.pathname? Vil bare at en meny-fane skal vises aktiv om gangen.